### ISSUE TITLE
About the metrics on Weibo datasets

### ISSUE THREAD

**Issue by adamas-v** [Lingzhi-WANG/Quotation-Recommendation#5]
HI ,thank a lot for your works, I success re-implement the map(31.8) on datasets Reddit ,but i get only map 32.2 for Weibo which should be 34.9 ,could U help me find out the reason. i set --model_name ncf_query --turn_encoder transformer for the training

**Comment by Lingzhi-WANG** [comment]
You should try more hyperparameters (learning, batch size, dropout, L2 weight, and so on). The model name and encoder name are right.

**Comment by adamas-v** [comment]
OK thanks a lot for you reply
